color.lisp

```diff
@@ -195,7 +195,7 @@ If COLOR isn't a colorcode a list containing COLOR is returned."
                     (screen-color-map-bright screen)
                     (screen-color-map-normal screen))
                 color))
-        (t (first (multiple-value-list (alloc-color screen color))))))
+        (t (nth-value 0 (alloc-color screen color)))))
```

Comment:
```suggestion
        (t (alloc-color screen color))))
```
Wouldn't this work just the same? Or we want to avoid returning multiple values from the function?

Reply:
> I assumed that wanting to avoid returning multiple values was intentional. 

It was probably intentional, but that intention is likely to be misguided. Unless calling m-v-b or m-v-l the caller won't see the additional values. But you made the right call by preserving the semantics when rewriting code that doesn't have tests.